docs/quickstart.rst

```diff
@@ -0,0 +1,77 @@
+.. highlight:: shell
+
+==========
+Quickstart
+==========
+
+
+Importing dask-image
+--------------------
+Import dask image is with an underscore, like this example:
+
+.. code-block:: python
+
+    import dask_image.imread
+    import dask_image.ndfilters
+
+
+Dask Examples
+-------------
+We highly recommend checking out the dask-image-quickstart.ipynb notebook 
+(and any other dask-image example notebooks) at the dask-examples repository:
+
+https://github.com/dask/dask-examples
+
+You can find the dask-image quickstart notebook in the `applications` folder
+of this repository. All the example notebooks are available to launch with
+mybinder and test out interactively.
+
+
+An Even Quicker Start
+---------------------
+
+You can read files stored on disk into a dask array
+by passing the filename, or regex matching multiple filenames
+into `imread()`.
+
+.. code-block:: python
+
+    filename_pattern = 'path/to/image-*.png'
+    images = dask_image.imread.imread(filename_pattern)
+
+If your images are parts of a much larger image, 
+dask can stack, concatenate or block chunks together:
+http://docs.dask.org/en/latest/array-stack.html
+
+
+Calling dask-image functions is also easy.
+.. code-block:: python
+
+    import dask_image.ndfilters
+    blurred_image = dask_image.ndfilters.gaussian_filter(images, sigma=10)
+
+
+If you have a function that accepts input numpy arrays,
+it is also likely to accept dask arrays without any modification. 
+
+.. code-block:: python
+
+    result = my_function(images)
+
+Remember that dask computations are lazy. 
+That's a great thing to have, most of the time.
+If you need to force dask to run the computation, use the `compute()` method.
```

Comment:
As this image data could be larger than memory, we should be somewhat careful recommending compute. Maybe it is worth noting some other options like `persist` and `store`. This is probably a situation where we can't go into too much detail, but we could have a brief intro followed by a link to appropriate Dask Array documented (e.g. maybe [this]( http://docs.dask.org/en/latest/array-creation.html#store-dask-arrays )). What do you think?

Reply:
Good point. I think I'll remove this bit altogether for now.